tests/python/compiler/test_top_level1.py

```diff
@@ -56,6 +127,19 @@ def test_log():
         y_np = np.log(data)
         np.testing.assert_allclose(out.asnumpy(), y_np, atol=1e-5, rtol=1e-5)
 
+        # backward
+        graph._set_symbol_list_attr('grad_ys', y)
```

Comment:
Consider add a python function for this

Reply:
I'll work on this to add creating backward graph for both c_api and python api.